migrations/2024-08-17-152408_add-yank-message/up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE versions ADD COLUMN yank_message VARCHAR(255);
```

Comment:
https://stackoverflow.com/a/4849030/1478093 seems to suggest that using `varchar(n)` could be problematic if we want to change the limit at some point:

> varchar(n) – it's problematic to change the limit in live environment (requires exclusive lock while altering table)

https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_varchar.28n.29_by_default also seems to discourage usage of `varchar(n)` in favor of `text`.

Reply:
Make sense. Updated.